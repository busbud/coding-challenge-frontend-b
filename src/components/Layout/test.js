import React from 'react';
import renderer from 'react-test-renderer';

import Component from './';

describe('Component', () => {
	it('should render', () => {
		const wrapper = renderer.create(<Component columns={['hey']} />).toJSON();
		expect(wrapper).toMatchSnapshot();
	});
});
