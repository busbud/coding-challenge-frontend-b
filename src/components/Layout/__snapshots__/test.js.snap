// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render 1`] = `
<div
  className="column"
>
  hey
</div>
`;
